### PR TITLE
feeds http_proxy

### PR DESCRIPTION
**Pull request by jinusean**
### Description of code - what bug does this fix / what feature does this add?

- [x] - Tested
- [x] - Changelog updated
- [x] - Tests run and pass
- [x] - Flake8 run and all errors/warnings resolved
- [x] - Contributors file updated (optional)

Enables a `http_proxy` param in Feed that is used by HttpProxy/HttpPoll when making GET calls.

### CODE REVIEW

**Comment by jinusean** [comment]
Should aid in overcoming issues described in https://github.com/bmoscon/cryptostore/issues/156

**Review by bmoscon — CHANGES_REQUESTED** [review]
it looks like you've added a new library requirement, YARL. I'm not opposed to adding libraries if they are used, but if you are using it solely for type hinting - is it really necessary?

File: cryptofeed/feed.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import logging
 import os
 from typing import Dict, Tuple, Callable, Union, List
+from aiohttp.typedefs import StrOrURL
```
Comment (jinusean) [inline cryptofeed/feed.py:12]: Replaced YARL with aiohttp's typedef

File: cryptofeed/connection.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 import websockets
 import requests
 from yapic import json as json_parser
-
+from aiohttp.typedefs import StrOrURL
```
Comment (jinusean) [inline cryptofeed/connection.py:18]: Replaced YARL with aiohttp's typedef

**Comment by jinusean** [comment]
YARL was used by aiohttp but using the latter's type definitions is probably better.

Thanks!

**Review by bmoscon — APPROVED** [review]

**Pull request merged**